translator.py

```
@@ -300,7 +330,7 @@ def translate_text(self, text: str, source_lang: str, target_lang: str) -> str:
     
     def translate_srt(self, input_path: str, source_lang: str, target_lang: str, 
                      method: str = "google", video_path: str = None, 
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)